send_sms.rb

```diff
@@ -0,0 +1,15 @@
+# Download the twilio-ruby library from twilio.com/docs/libraries/ruby
```

Comment:
This is a perfectly good procedural implementation of sending messages and there's definitely a place for procedural programs / scripts.  Could you refactor this to use a new class instead?